### PR TITLE
infix: init at 1.2

### PR DESCRIPTION
**Pull request by pasunboneleve**
###### Motivation for this change

New package.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/infix/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+buildPythonPackage rec {
+  pname = "infix";
+  version = "1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "a1bfdcf875bc072f41e426d0673f2e3017750743bb90cc725fffb292eb09648c";
+  };
+
+  doCheck = true;
```
Comment (FRidh) [inline pkgs/development/python-modules/infix/default.nix]: not needed

File: pkgs/development/python-modules/pytest-testmon/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "df00594e55f8f8f826e0e345dc23863ebac066eb749f8229c515a0373669c5bb";
+  };
+
+  doCheck = false; # tests written in Python 2; library compiles and runs properly otherwise.
+  buildInputs = with pkgs; [ glibcLocales ];
+  propagatedBuildInputs = with pythonPackages; [ pytest coverage ];
+  preConfigure = ''
```
Comment (FRidh) [inline pkgs/development/python-modules/pytest-testmon/default.nix]: I don't think this is needed anymore

File: pkgs/development/python-modules/pytest-watch/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  pname = "pytest-watch";
+  version = "4.2.0";
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06136f03d5b361718b8d0d234042f7b2f203910d8568f63df2f866b547b3d4b9";
+  };
+
+  doCheck = true;
```
Comment (FRidh) [inline pkgs/development/python-modules/pytest-watch/default.nix]: not needed

File: pkgs/development/python-modules/pytest-testmon/default.nix [hunk 4]
```diff
@@ -0,0 +1,33 @@
+{ pkgs
```
Comment (FRidh) [inline pkgs/development/python-modules/pytest-testmon/default.nix]: don't pass in `pkgs`. Pass in actual packages instead

File: pkgs/development/python-modules/pytest-testmon/default.nix [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+, lib
+, pythonPackages
```
Comment (FRidh) [inline pkgs/development/python-modules/pytest-testmon/default.nix]: don't pass in `pythonPackages`. Pass in actual packages instead.

File: pkgs/development/python-modules/pytest-watch/default.nix [hunk 6]
```diff
@@ -0,0 +1,29 @@
+{ lib
+, pkgs
```
Comment (FRidh) [inline pkgs/development/python-modules/pytest-watch/default.nix]: as before

File: pkgs/development/python-modules/pytest-watch/default.nix [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+, pythonPackages
```
Comment (FRidh) [inline pkgs/development/python-modules/pytest-watch/default.nix]: as before

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by dmvianna** [comment]
Sorry @FRidh, this PR is a duplicate, and I could not find it before you spent time reviewing it. The packages being submitted here were reviewed in #63075 and #63007.

Thanks for your effort.

**Pull request closed without merging**